src/renderer/features/governance/aggregates/unlock.ts

```diff
@@ -28,22 +30,28 @@ sample({
   target: stepChanged,
 });
 
+sample({
+  clock: unlockFormStarted,
+  fn: () => Step.SELECT,
+  target: stepChanged,
+});
+
 sample({
   clock: stepChanged,
   target: $step,
 });
 
 sample({
-  clock: unlockConfirm,
-  source: {
-    unlockableClaims: unlockModel.$claimSchedule.map((c) =>
-      c.filter((claim) => claim.type === UnlockChunkType.CLAIMABLE),
-    ),
-    totalUnlock: unlockModel.$totalUnlock,
-  },
-  filter: ({ totalUnlock }) => !totalUnlock.isZero(),
-  fn: ({ unlockableClaims, totalUnlock }) => ({
-    event: { unlockableClaims, amount: totalUnlock.toString() },
+  clock: unlockFormStarted,
+  source: unlockModel.$claimSchedule,
+  fn: (claims) => claims.filter((claim) => claim.type === UnlockChunkType.CLAIMABLE) as ClaimChunkWithAddress[],
```

Comment:
This case is covered in typescript 5.5, 
```suggestion
  fn: (claims) => claims.filter((claim) => claim.type === UnlockChunkType.CLAIMABLE),
```

Reply:
Hm, without this -  types don't match and it still believes that I'm trying to pass UnlockChunk[] not ClaimChunkWithAddress[]